Tests/EndToEndTests/EndToEndTests.swift

```diff
@@ -0,0 +1,50 @@
+import class Foundation.Bundle
+import XCTest
+
+final class EndToEndTests: XCTestCase {
+  func testCSS() throws {
+    // Some of the APIs that we use below are available in macOS 10.13 and above.
+    guard #available(macOS 10.13, *) else {
+      return
+    }
+
+    let process = Process()
+    process.executableURL = productsDirectory.appendingPathComponent("swift-doc")
+    process.arguments = ["generate", "--module-name", "SwiftDoc", "--format", "html", "Sources"]
+
+    let pipe = Pipe()
+    process.standardOutput = pipe
+
+    try process.run()
+    process.waitUntilExit()
```

Comment:
```suggestion
    let process = Process()
    process.executableURL = productsDirectory.appendingPathComponent("swift-doc")
    process.arguments = ["generate", "--module-name", "SwiftDoc", "--format", "html", "Sources"]

    let pipe = Pipe()
    process.standardOutput = pipe

    if #available(OSX 10.13, *) {
      try process.run()
    } else {
      process.launch()
    }
    process.waitUntilExit()
```

Reply:
As far as I can tell, the only macOS 10.13 API here is `process.run`, correct? If so, could we tailor the availability check to conditionally use `process.launch` instead where `run` isn't available?